python/paddle/fluid/incubate/fleet/parameter_server/pslib/__init__.py

```diff
@@ -28,63 +28,56 @@ class PSLib(Fleet):
     def __init__(self):
         super(PSLib, self).__init__(Mode.PSLIB)
         self._opt_info = None
-        self.local_ip_ = 0
+        self._local_ip = 0
         self._fleet_ptr = None
+        self._main_programs = []
+        self._scopes = []
 
-    def init(self, role_maker=None):
-        super(PSLib, self).init(MPISymetricRoleMaker())
+    def init(self, executor, role_maker=None):
+        super(PSLib, self).init(executor, MPISymetricRoleMaker())
         self._fleet_ptr = fluid.core.Fleet()
 
-    def init_worker(self, executor):
-        pass
-
-    def run_worker(self, executor, main_program=None):
+    def init_worker(self):
         """
         init_worker(): will be called by user. When a user knows current process is_server(), he/she
                     should call init_worker() to initialize global information about worker and connect
                     worker with pserver. You should run startup program before init_worker.
 
         Args:
-            programs(Program|list): a Program or a list of Programs
-            scopes(Scope|list): a Scope or  a list of Scopes, default None.
+            executor(Executor): The executor to run for init server.
+            programs(Program|None): The program that need to run.
         """
```

Comment:
what if a user uses  more than one scope , each for a specific main_program.  
in this case,  init_worker no longer supports this.

Reply:
oh，these scopes are passed to fleet in minimize，ignore the above.